Jenkinsfile

```diff
@@ -43,5 +55,39 @@ pipeline{
         '''
       }
     }
+    stage('regressions'){
+      when{
+        allOf{
+          branch 'develop'
+          expression {return params.regressions}
+        }
+      }
+      parallel{
+        stage('11-nightly-java11'){
+          agent {label 'devExpLinuxPool'}
+          steps{
+            runtests('Latest','11','JAVA11_HOME_DIR')
+          }
+        }
+        stage('11-nightly-java17'){
+          agent {label 'devExpLinuxPool'}
+          steps{
+            runtests('Latest','11','JAVA17_HOME_DIR')
+          }
+        }
+        stage('10.0-9.5-java11'){
+          agent {label 'devExpLinuxPool'}
+          steps{
+            runtests('Release','10.0-9.5','JAVA11_HOME_DIR')
+          }
+        }
+        stage('10.0-9.5-nightly-java17'){
```

Comment:
@SameeraPriyathamTadikonda  No need for this one - we just need one job that runs on Java 17, that's all.